src/richie/apps/core/templates/richie/base.html

```diff
@@ -147,7 +147,7 @@
                         </div>
                         <div class="body-footer__brand">
                             {% block body_footer_brand %}
-                                <a href="/">
+                                <a href="/" title="{{ SITE.name }}">
```

Comment:
Hey,

Instead of the `title` attribute here, I'd rather fill the img `alt` attribute with the site name. Filling the alt is the primary way to add alt text on a `img`. The `title`, accessibility-wise, is more of a fallback. Here it would have the additional bonus of showing a tooltip to help mouse users understand if needed, but it's not really necessary since the logo is self-explanatory.

Reply:
The only issue here is that, in our case (as an example) we use css inline SVG's to display our logo. But we've changed the  template. That was the reason for the `title` tag, but yeah, the `alt` is a much better overall solution, specially for the older browser. I'll fix this as well. Thanks.